pkg/bot.go

```diff
@@ -149,24 +160,25 @@ func authAdminAPI(k *kubernetes.Clientset, authSVCLabels string) (pbAdmin.Admini
 	return pbAdmin.NewAdministrationClient(conn), nil
 }
 
-func InitBot(kubeconfig *restclient.Config, annotation, siteSuffix string, authSVCLabels string, stopCh <-chan struct{}) (Bot, error) {
-	scs, err := siteclientset.NewForConfig(kubeconfig)
+func InitBot(bc Config) (Bot, error) {
```

Comment:
Is this called by another service?  Are we mounting the AnalyticsKey via secret?

Reply:
this is called by github-operator and gitlab-webhook-server. Yes, my plan was to mount the key via `SealedSecret` in a state repo, it has to be different for prod and non-prod.

Reply:
Thanks.  Theres a bit of oddness required in how the CLI bakes this in I am sure you noticed, perhaps required for CLI's distributed to our users.